modules/kwin.nix

```diff
@@ -79,18 +85,94 @@ in
       shakeCursor.enable = mkOption {
         type = with types; nullOr bool;
         default = null;
-        example = true;
-        description = "Enable the shake cursor effect (plasma 6 only).";
+        description = "Enable the shake cursor effect.";
       };
-    };
-
-    virtualDesktops = {
-      animation = mkOption {
-        type = with types; nullOr (enum [ "slide" "fade" ]);
+      translucency.enable = mkOption {
+        type = with types; nullOr bool;
+        default = null;
+        description = "Make windows translucent under different conditions.";
+      };
+      minimization = {
+        animation = mkOption {
+          type = with types; nullOr (enum [ "squash" "magiclamp" ]);
+          default = null;
+          example = "magiclamp";
+          description = "The effect when windows are minimized.";
+        };
+        duration = mkOption {
+          type = with types; nullOr ints.positive;
+          default = null;
+          example = 50;
+          description = ''
+            The duration of the minimization effect in milliseconds. Only
+            available when the minimization effect is magic lamp.
+          '';
+        };
+      };
+      wobblyWindows.enable = mkOption {
```

Comment:
I know wobbly effects has configurations, is that intentionality that it is not configurations here?

Reply:
It's coming, but I focused on adding options for enabling first and I'll probably add options for configuring the different effects later.